browser/ui/brave_browser_command_controller.cc

```diff
@@ -360,6 +360,7 @@ void BraveBrowserCommandController::UpdateCommandForAIChat() {
   bool command_enabled = (sidebar::CanUseSidebar(&*browser_) &&
                           ai_chat::IsAllowedForContext(browser_->profile()));
   UpdateCommandEnabled(IDC_TOGGLE_AI_CHAT, command_enabled);
+  UpdateCommandEnabled(IDC_OPEN_FULL_PAGE_CHAT, true);
```

Comment:
should only be enabled if AIChatHistory feature flag is enabled and AI Chat is allowed in the context